docker/Dockerfile

```diff
@@ -3,38 +3,226 @@
 #
 # Base image Ubuntu 18.04
 
-FROM ubuntu:18.04
 
-ENV DEBIAN_FRONTEND=noninteractive
-ENV MEDIA_SDK_VER=18.4.0
-ARG IMG_APP_PATH=/app_data/
-ENV APP_PATH=${IMG_APP_PATH}
```

Comment:
I would prefer we keep the APP_PATH environment variable so that build_server.sh knows where to install the user created application. Rather than having the user specifying the APP_PATH. Otherwise the user needs to know the directory structure of the owt image to make sure they don't overwrite something they shouldn't.

Reply:
Yes, it's better to keep the APP_PATH environment variable, will update the Dockerfile